examples/multiuser-scenario/test/scenario.js

```diff
@@ -0,0 +1,89 @@
+const createTestCafe = require('testcafe');
```

Comment:
Let's move it to `<example root>/index.js` and specify it as the `main` module in `package.json`.